app/bag_of_holding.py

```diff
@@ -233,7 +188,10 @@ def _getWeight(items):
     return round(total_weight, 1)
 
 def _convertDefault(coins, default="Gold"):
-    default_value = Coin.query.filter_by(_name=default).first()._value
+    default_value = Coin.query.filter_by(_name=default).first()
+
+    # TODO Make the application stateless
+    default_value = default_value._value if default_value else 1
```

Comment:
This is a null pointer exception waiting to happen, if the `Coins` table is empty (as mine was) calling `_value` will throw an exception since `.first()` will return `None`.

This is ternary is a bad workaround, the best solution would be either:
1. Have a migration management tool that populates the Coins table with a default value.
2. To not use the database state and instead use a static value or configurable value (I prefer this way)